modules/dfpAdServerVideo.js

```diff
@@ -249,6 +254,68 @@ function getCustParams(bid, options, urlCustParams) {
   return encodedParams;
 }
 
+async function getVastForLocallyCachedBids(gamVastWrapper, localCacheMap) {
+  try {
+    const xmlUtil = XMLUtil();
+    const xmlDoc = xmlUtil.parse(gamVastWrapper);
+    const vastAdTagUriElement = xmlDoc.querySelectorAll(VAST_TAG_URI_TAGNAME)[0];
+
+    if (!vastAdTagUriElement || !vastAdTagUriElement.textContent) {
+      return gamVastWrapper;
+    }
+
+    const uuidExp = new RegExp(`.*(?:([A-Fa-f0-9]{8}-(?:[A-Fa-f0-9]{4}-){3}[A-Fa-f0-9]{12}).*)+`, 'gi');
```

Comment:
`[A-Fa-f0-9]{8}-(?:[A-Fa-f0-9]{4}-){3}[A-Fa-f0-9]{12}` should work and with `matchAll` give you all UUID candidates (since you're also giving it the 'g' global flag). if you match everything else (wrapping it in `.*(...).*`) it will match only once and give you the one capturing group within.